src/components/PageTitle.tsx

```diff
@@ -32,5 +36,15 @@ export const PageTitle: React.FC<PageTitleProps> = ({ title, subtitle, onClick }
                 </StyledText>
             </>
         )}
+
+        {info && (
+            <>
+                {' '}
```

Comment:
Use css offset for sibling elements with margin-left instead of space character 🧐